docs/user_guide/bank/community_product_drives.md

```diff
@@ -1 +1,25 @@
-Not yet written
\ No newline at end of file
+# Product Drive Participants
+
+You may have organized a product drive in your community and would like to manage the participants.
+
+To manage product drive participants, click on " Product Drive Participants" under the "Community" section.
+
+![Product Drive Participants](images/community/product_drive/product_drive_page.jpg)
+
+If you have done product drives before, the page will be populated with the participants' details. 
+
+To add a new participant, click on the "+ New Product Drive Participant" button, add their details including the business name, contact name, phone, email, address and comments.
+
+![Add Participant](images/community/product_drive/add_paticipant.jpg)
+
+After saving the participant's details there will be a new row in the Product Drive Participants page.
+
+Click the "Edit" button to edit the participant's details. 
+
+Click on "View" for more details about the product drive participant which also shows the date of donation, volume, and donations details.
+
+![Participant Details](images/community/product_drive/participant_details.jpg)
+
+You can export all the participants details by clicking on the "Export Product Drive Paticipants" button.
```

Comment:
You can export the participants by...    


(Reason:  we aren't providing all the details in the export -- though I think it might be a good idea if we put the comments in, we aren't yet)